evalai/utils/challenges.py

```diff
@@ -194,6 +194,47 @@ def display_participated_or_hosted_challenges(is_host=False, is_participant=Fals
             echo("Sorry, no challenges found!")
 
 
+def pretty_print_challenge_details(challenge):
+    table = BeautifulTable(max_width=210)
+    attributes = ["description", "submission_guidelines", "evaluation_details", "terms_and_conditions"]
+    table.column_headers = ["Title", "Start Date", "End Date", "Description", "Submission Guidelines",
```

Comment:
I think we can remove `Title` from here as it gets displayed when a user lists all challenges. I am asking this as it is distorting the complete table.